README.md

```diff
@@ -1,2 +1,29 @@
-# reana-jupyterlab-extension
-Reana JupyterLab plugin
+# Reana JupyterLab Extension
+Reana JupyterLab plugin provides a set of tools to interact with the [Reana](https://reanahub.io/) workflow management system from within JupyterLab. 
+
+**This project is currently in development and is subject to change.**
+
+## Requirements
+- JupyterLab >= 4
```

Comment:
Specify on the README a JLab version >=4 and <5, please.

Reply:
Fixed